auth/auth-mock/src/main/java/me/nalab/auth/mock/interceptor/MockAuthInterceptor.java

```diff
@@ -24,8 +28,16 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
 	}
 
 	private boolean isExcludedURI(HttpServletRequest httpServletRequest) {
-		return httpServletRequest.getMethod().equals("POST") && httpServletRequest.getRequestURI()
-			.contains("/v1/feedbacks");
+		String httpMethod = httpServletRequest.getMethod();
+		String requestURI = httpServletRequest.getRequestURI();
+
+		for (String[] excludedURI: EXCLUDED_URI_LIST) {
```

Comment:
```suggestion
		for (String[] excludedURI : EXCLUDED_URI_LIST) {
```

: 이거 한칸 띠우는 컨벤션이 있었나..?